packages/wonder-blocks-popover/components/popover-dialog.js

```diff
@@ -0,0 +1,124 @@
+// @flow
```

Comment:
This component is a lite version of `TooltipBubble`. I had to create a new component b/c of the different constraints between both packages. 

Main difference is that `TooltipBubble` handles hover states and `PopoverDialog` doesn't need to handle any states at all (for now). Also, `PopoverDialog` needs to coordinate different background colors for the content and tail components.

Reply:
I see, it's been added below.  Thank you.